### DISCUSSION TITLE
WingetUI 3.0 Alpha 1

### DISCUSSION THREAD

**Discussion by marticliment** [Devolutions/UniGetUI#1803] [Show and tell]
![](https://img.shields.io/github/downloads/marticliment/wingetui/3.0.0-alpha1/total?style=for-the-badge&color=red)

# Main changes since Alpha 0

 - Addes Importing and exporting packages feature, now renamed to Package Bundles
   - Bundles can now be edited
   - Packages can now be added one (or multiple) at a time. No need to bulk-select all of them
   - Bundles can be exported as XML (in addition to already existing JSON and XAML) too.
 - Package backups do work now
 - Sources can be added and removed
 - Package details will show.
 - Fix crashes and improve overall program

<hr /><em>This discussion was created from the release <a href='https://github.com/marticliment/WingetUI/releases/tag/3.0.0-alpha1'>WingetUI 3.0 Alpha 1</a>.</em>

**Comment by iLL888** [comment]
Hello,
Nice work and I enjoy using WingetUI, here is some feedback based on my experience. 

- When software fails to install because the hash does not match, adding a option 'Ignore hash' next to 'Close' would be useful. 
- When a install fails for any other unknown reason, perhaps a 'retry' option would be useful.
- When software is successfully installed, the success queue bar auto disappears. This is fine, however if your mouse is hovered over this queued item, it should not disappear. Many times have I went to press the "Close" button for it to disappear and I accidentally press the cancel button for the next queue item.
- The ability to change the order of the queue items by adding the ability to move item up/down, or move to top.
- The ability to pause all queue items. 
- The ability to remember queue items after you exit WingetUI.  Sometimes, the PC may need to reboot in the middle of the queue. However , if you close the application with items in queue, you will lose the entire queue on exit. Same if the PC locks up or you close the application for any reason. 
- Would be useful to have a button to easily open the Temp directory where the installation files are downloaded.
- Chinese characters dont seem to work in the new Alpha.
- When installing items, the icon status (in the search) does not change.  For example, you cant see whats installing, already installed or in queue. 
- In the settings there is the Winget Sources, an ability to reset these sources would be useful. (or the option to disable a source)
- For NPM Not Found - Add link or resource for more information.
- For .NET Tool Not Found - Add link or resource for more information.
- The progress bar for search (blue horizontal bar) never stops moving even when the search has completed. 
- Search Mode (sidebar) does not have a default option selected.
- I liked the ability to make the download queue screen larger. (not there now)
- Some search terms do not work, for example if I search for "game" it will not display any results. But if I search for "gam" or "gamer" then results are displayed.  This happens with various search terms.
- HTML/Script appears at the very top of the settings screen. (ez_ab_test = 'mod1')
- Personally I prefer limiting the search to 2 characters instead of 3, as this makes it more usable for Chinese users.  For example, Baidu is two characters (百度).  In addition, if someone was to search for AI, (for AI software) they would need to add more context to the search.

**Reply by marticliment** [reply]
I will take your suggestions into account

**Comment by itayasaf1990xd** [comment]
Hey,
Are you going to rebrand wingetUI with update version 3?

BTW good update I am so glad you moved to .NET thanks you for your hard work!

**Reply by marticliment** [reply]
Possibly yes, I still have to decide it but most likely yes. I will let you know on the discussion #1561
